docs/api/clojure/module.md

```diff
@@ -0,0 +1,235 @@
+
+
+## Saving and Loading
+
+To save the module parameters in each training epoch, use a `checkpoint` function:
+
+```clojure
+(let [save-prefix "my-model"]
+  (doseq [epoch-num (range 3)]
+    (mx-io/do-batches train-data (fn [batch
+                                          ;; do something
+]))
+    (m/save-checkpoint mod {:prefix save-prefix :epoch epoch-num :save-opt-states true})))
+
+;; INFO  org.apache.mxnet.module.Module: Saved checkpoint to my-model-0000.params
+;; INFO  org.apache.mxnet.module.Module: Saved optimizer state to my-model-0000.states
+;; INFO  org.apache.mxnet.module.Module: Saved checkpoint to my-model-0001.params
+;; INFO  org.apache.mxnet.module.Module: Saved optimizer state to my-model-0001.states
+;; INFO  org.apache.mxnet.module.Module: Saved checkpoint to my-model-0002.params
+;; INFO  org.apache.mxnet.module.Module: Saved optimizer state to my-model-0002.states
+
+```
+
+To load the saved module parameters, call the `load-checkpoint` function:
+
+```clojure
+(def new-mod (m/load-checkpoint {:prefix "my-model" :epoch 1 :load-optimizer-states true}))
+
+new-mod ;=> #object[org.apache.mxnet.module.Module 0x5304d0f4 "org.apache.mxnet.module.Module@5304d0f4"]
+```
+
+To initialize parameters, Bind the symbols to construct executors first with bind function. Then, initialize the parameters and auxiliary states by calling `init-params` function.
+
+```clojure
+(-> new-mod
+    (m/bind {:data-shapes (mx-io/provide-data train-data) :label-shapes (mx-io/provide-label train-data)})
+    (m/init-params))
+```
+
+To get current parameters, use `params`
+
+```clojure
+
+(let [[arg-params aux-params] (m/params new-mod)]
+  {:arg-params arg-params
+   :aux-params aux-params})
+
+;; {:arg-params
+;;  {"fc3_bias"
+;;   #object[org.apache.mxnet.NDArray 0x39adc3b0 "org.apache.mxnet.NDArray@49caf426"],
+;;   "fc2_weight"
+;;   #object[org.apache.mxnet.NDArray 0x25baf623 "org.apache.mxnet.NDArray@a6c8f9ac"],
+;;   "fc1_bias"
+;;   #object[org.apache.mxnet.NDArray 0x6e089973 "org.apache.mxnet.NDArray@9f91d6eb"],
+;;   "fc3_weight"
+;;   #object[org.apache.mxnet.NDArray 0x756fd109 "org.apache.mxnet.NDArray@2dd0fe3c"],
+;;   "fc2_bias"
+;;   #object[org.apache.mxnet.NDArray 0x1dc69c8b "org.apache.mxnet.NDArray@d128f73d"],
+;;   "fc1_weight"
+;;   #object[org.apache.mxnet.NDArray 0x20abc769 "org.apache.mxnet.NDArray@b8e1c5e8"]},
+;;  :aux-params {}}
+
+```
+
+To assign parameter and aux state values, use `set-params` function.
+
+```clojure
+(m/set-params new-mod {:arg-params (m/arg-params new-mod) :aux-params (m/aux-params new-mod)})
+;=> #object[org.apache.mxnet.module.Module 0x5304d0f4 "org.apache.mxnet.module.Module@5304d0f4"]
+```
+
+To resume training from a saved checkpoint, instead of calling `set-params`, directly call `fit`, passing the loaded parameters, so that `fit` knows to start from those parameters instead of initializing randomly
```

Comment:
To resume training from a saved checkpoint, pass the loaded parameters to the `fit` function. This will prevent `fit` from initializing randomly. 
(not sure if you still need to bring up `set-params` here or not.)